modules/rosa-hcp-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-hcp-shared-vpc-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sharing-vpc-creation-and-sharing_{context}"]
+= Step One - VPC Owner: Configuring a VPC to share within your AWS organization
+
+You can share subnets within a configured VPC with another AWS user account if that account is within your current AWS organization.
+
+image::522-shared-vpc-step-1.png[]
+.Procedure
+
+. Create or modify a VPC to your specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console].
++
+. Create the `Route 53 role`.
```

Comment:
Needs to include some warning block to mention that "if you wish to keep the hosted zones in the same account of the cluster resource the route 53 role needs to be created in the cluster owner account instead"